Documentation/kubernetes-on-vagrant.md

```diff
@@ -60,15 +60,25 @@ Next, simply run `vagrant up` and wait for the command to succeed.
 Once Vagrant is finished booting and provisioning your machine, your cluster is good to go.
 
 ## Configure kubectl
+You can choose from one of the two following options.
 
-Configure your local Kubernetes client using the following commands:
+1. **Use a custom KUBECONFIG path**
 
-```sh
-$ kubectl config set-cluster vagrant --server=https://172.17.4.101:443 --certificate-authority=${PWD}/ssl/ca.pem
-$ kubectl config set-credentials vagrant-admin --certificate-authority=${PWD}/ssl/ca.pem --client-key=${PWD}/ssl/admin-key.pem --client-certificate=${PWD}/ssl/admin.pem
-$ kubectl config set-context vagrant --cluster=vagrant --user=vagrant-admin
-$ kubectl config use-context vagrant
-```
+	```sh
+	export KUBECONFIG="${KUBECONFIG}:$(pwd)/kubeconfig"
+	kubectl config use-context vagrant-multi
+	```
+
+2. **Update the local-user kubeconfig**
+
+	Configure your local Kubernetes client using the following commands:
+
+	```sh
+	$ kubectl config set-cluster vagrant-multi --server=https://172.17.4.101:443 --certificate-authority=${PWD}/ssl/ca.pem
+	$ kubectl config set-credentials vagrant-admin --certificate-authority=${PWD}/ssl/ca.pem --client-key=${PWD}/ssl/admin-key.pem --client-certificate=${PWD}/ssl/admin.pem
```

Comment:
This will overwrite the `vagrant-admin` set in the single node cluster if someone has already followed those steps. If the namespacing is meant to make this easier to switch between (e.g. `kubect use-context vagrant-[single|multi]`) then we need to namespace this too: `vagrant-multi-admin`